src/wikigraph.cc

```diff
@@ -1,21 +1,26 @@
 #include "wikigraph.hpp"
-#include "utilities.hpp"
-#include <unordered_map>
+
 #include <fstream>
+#include <queue>
+#include <unordered_map>
+
+#include "utilities.hpp"
 
 // TODO: Construct from File
 WikiGraph::WikiGraph(const std::string& file_name) {
   // Reads in file stream.
   // Populates map.
 
   // open file
-  // get a line; decode the first item, decode the second item, add the second to adjlist of firsts
+  // get a line; decode the first item, decode the second item, add the second
+  // to adjlist of firsts
   std::string file = file_to_string(file_name);
   std::vector<std::string> lines;
   SplitString(file, '\n', lines);
-  lines.pop_back(); // our dataset has a line of empty space that we can discard
+  lines
+      .pop_back();  // our dataset has a line of empty space that we can discard
```

Comment:
i personally despise this style change and we should merge the third constructor PR instead of doing this

Reply:
must've been a weird linter issue. i didn't make these changes manually